pkg/roll/run_update.go

```diff
@@ -402,9 +403,11 @@ func (u *update) countHealthy(id rcf.ID, checks map[types.NodeName]health.Result
 		node := pod.Node
 		// TODO: is reality checking an rc-layer concern?
 		realManifest, _, err := u.kps.Pod(kp.REALITY_TREE, node, rcFields.Manifest.ID())
-		if err != nil {
+		if err != nil && err != pods.NoCurrentManifest {
 			return ret, err
 		}
+
+		// even if realManifest is nil, we'll get empty string for SHA which is fine for comparison purposes.
```

Comment:
`realManifest` has type `manifest.Manifest`, an interface, and you can't call any method on a `nil` interface value.